test/integration-tests/RemoteAdapterAdd.t.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.9;
+
+/// library imports
+import {Vm} from "forge-std/Test.sol";
+
+/// local imports
+import "test/Setup.t.sol";
+
+import {MultiMessageSender} from "src/MultiMessageSender.sol";
+import {MultiMessageReceiver} from "src/MultiMessageReceiver.sol";
+import {Error} from "src/libraries/Error.sol";
+import {GovernanceTimelock} from "src/controllers/GovernanceTimelock.sol";
+
+/// @dev scenario: admin updates config on dst chain using message from source chain
+contract RemoteAdapterAdd is Setup {
+    /// @dev intializes the setup
+    function setUp() public override {
+        super.setUp();
+    }
+
+    /// @dev just sends a message
+    function test_remoteAddReceiverAdapter() public {
```

Comment:
I don't think this test works. 
I believe the Axelar adapter is already added to the remote chain as part of the setup step, so the [postconditions that are checked at the end of the function](https://github.com/MultiMessageAggregation/multibridge/pull/57/files#diff-64abc34aab5f7cb10df7cc11290e8dcc27d5088ae40c8e7df0a91486faa5c720R66-R69) that check whether an adapter has been successfully added, would still pass if executed at the start of this function.

Reply:
so is line 66 incorrect then? Instead of checking that the Axelar adapter is trusted, it should check that this random adapter is trusted right?